decode_morse.rb

```diff
@@ -0,0 +1,25 @@
+@decoder_hash = {
+  '.-' => 'A', '-...' => 'B', '-.-.' => 'C', '-..' => 'D', '.' => 'E', '..-.' => 'F', '--.' => 'G',
+  '....' => 'H', '..' => 'I', '.---' => 'J', '-.-' => 'K', '.-..' => 'L', '--' => 'M', '-.' => 'N',
+  '---' => 'O', '.--.' => 'P', '--.-' => 'Q', '.-.' => 'R', '...' => 'S', '-' => 'T', '..-' => 'U',
+  '...-' => 'V', '.--' => 'W', '-..-' => 'X', '-.--' => 'Y', '--..' => 'Z', '.----' => '1',
+  '..---' => '2', '...--' => '3', '....-' => '4', '.....' => '5', '-....' => '6', '--...' => '7',
+  '---..' => '8', '----.' => '9', '-----' => '0'
+}
+def decode_char(char)
+  @decoder_hash[char]
+end
+
+def word_decoder(word)
+  word.split.map do |char|
+    decode_char(char)
+  end.join
+end
+
+def decode(morse_code)
+  morse_code.split('   ').map do |word|
+    word_decoder(word)
+  end.join(' ')
+end
+
+puts decode('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')

```

Comment:
### Great job 👍🏻 
```
puts decode('.-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...')

puts decode_char('.-') 
puts word_decoder('-- -.--')
puts decode("-- -.--   -. .- -- .") 
puts decode(".-   -... --- -..-   ..-. ..- .-.. .-..    --- ..-.    .-. ..- -... .. . ...")
```

![18](https://user-images.githubusercontent.com/79058364/185617630-9279c456-ec9e-448d-a1cf-56d0cad7136a.png)

Reply:
Thank you very much @BenMukebo you are the best